plsc_ordered.py

```diff
@@ -412,9 +412,18 @@ def create(src, dst):
                     'description': ['All CO members'],
                     'displayName': [f'All Members of {vc[service][co_identifier]["name"]}']
                 }
+
+                # Add tags
                 if vc[service][co_identifier]["tags"]:
                     grp_entry['businessCategory'] = vc[service][co_identifier]["tags"]
 
+                # Add labeledURI
```

Comment:
DIt lijkt dubbel?  Zie regel 205

Reply:
Dat klopt, we hebben 'm ook twee keer nodig toch? Of wil je de labeledURI in de service laten zitten?